rust/src/router/transport.rs

```diff
@@ -522,13 +523,24 @@ pub(super) trait TransportImpl: TransportGeneric {
             }
         };
 
+        let computed_enable_nack: bool;
+
+        if let Some(flag) = enable_nack {
+            computed_enable_nack = flag;
+        } else if producer.kind() == MediaKind::Video {
+            computed_enable_nack = true;
+        } else {
+            computed_enable_nack = false;
+        }
```

Comment:
This would look more idiomatic in Rust:
```suggestion
        let enable_nack = enable_nack.unwrap_or(
            producer.kind() == MediaKind::Video
        );
```

It means you're extracting provided value if it is there, but if not using default value, which is true for video. And I'd probably inline it in `ortc::get_consumer_rtp_parameters` call since it is only necessary for `else` branch.

Reply:
Thanks. Done.